tests/errors.spec.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference path="mochainon.d.ts" />
```

Comment:
Again this is a leftover of my desperation to fix it. I wanted to keep this scoped to the tests only indeed, and it's impossible to import the typings otherwise if the module is already imported. But I'm fine with the current solution.